src/pages/jihong/Main/Aside/Aside.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './Aside.scss';
+import AsideStory from './Story/AsideStory';
+import STORY_LIST from './Story/storyData';
+import AsideInfo from './Info/AsideInfo';
+import INFO_LIST from './Info/infoData';
+import AsideFollow from './Follow/AsideFollow';
+import FOLLOW_LIST from './Follow/followData';
+
+const Aside = () => {
+  return (
+    <div className="aside">
+      <div className="asideProfile">
+        <img
+          className="profileImg3"
+          src="./images/jihong/jihong.jpeg"
```

Comment:
이미지 경로는 public 하위의 절대경로이기 때문에 . 안찍어도 됨!
```suggestion
          src="/images/jihong/jihong.jpeg"
```